docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -151,3 +139,56 @@ fed-minion          <none>              Ready
 **The cluster should be running! Launch a test pod.**
 
 You should have a functional cluster, check out [101](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/walkthrough/README.md)!
+
+**Run time addition/deletion of minions (or nodes).**
+
+* Create following minion.json file on kubernetes master node:
+
+```json
+{
+  "id": "fed-node1",
+  "kind": "Minion",
+  "apiVersion": "v1beta1",
+  "resources": {
+    "capacity": {
+      "cpu": 1,
+      "memory": 1073741824
+    },
+  },
+  "labels": {
+    "name": "fed-node1-label",
+  },
+}
+```
+
+Now create a minion object internally in your kubernetes cluster by running:
+
+```
+$ kubectl create -f minion.json
+
+$ kubectl get nodes
+NAME                LABELS              STATUS
+fed-node1           name=fed-node1-label     Unknown
```

Comment:
+1 on removal of --machines, but we should probably make note that any entry requires manual cleaning in the case of cluster changes.